apisix/http/router/radixtree_sni.lua

```diff
@@ -96,6 +96,10 @@ local function create_router(ssl_items)
     end
 
     core.log.info("route items: ", core.json.delay_encode(route_items, true))
+    -- for testing
```

Comment:
We can check the existing error log, we do not need this testing code.

![image](https://user-images.githubusercontent.com/6814606/87127159-0c0e6200-c2c0-11ea-8071-1efcfb12143a.png)

Reply:
the existing log is a large string, it's hard to match, and the snis are reverse.
so i thought adding a new info log is better ?

Reply:
ok, got it. we can keep it.